src/main/resources/db/migration/common/V1_476__new_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX event_event_id_idx ON darts."event" (event_id,message_id,event_text);
```

Comment:
Can you just run this one past Barry please and make sure he's happy for it to go in?